app/server/utils/user.py

```diff
@@ -222,14 +223,16 @@ def create_transfer_account_user(first_name=None, last_name=None, preferred_lang
         if current_app.config['AUTO_APPROVE_TRANSFER_ACCOUNTS'] and not is_self_sign_up:
             transfer_account.approve()
 
+    user.default_transfer_account_id = transfer_account.id
```

Comment:
this was copied over from my PR right? it'll break master because the attr is defined on my PR

Reply:
Has to have been because all I remember doing here was adding the methods to send messages.